vector/help/faq.mdx

```diff
@@ -47,10 +47,6 @@ title: FAQ
     
     We don’t support hybrid search at the moment. Although this is in our roadmap we still haven’t finalized the approach to help our users eliminate the need to have a secondary database for better search result. As we define our approach on implementing hybrid search and gather more user feedback, we will share this through our blog posts
     
-- **Can I filter by metadata?**
```

Comment:
We can still keep this question, and add a reference to the metadata filtering process.